package.json

```diff
@@ -49,10 +49,13 @@
     "truffle-flattener": "^1.2.10"
   },
   "dependencies": {
+    "@daostack/arc": "0.0.1-rc.8",
```

Comment:
Do we really have to import all their code? Is this not in a separate npm module? If so, what a pity.

Reply:
I couldn't find it anywhere else. It's only actually compiling the contracts that are used. The alternative would be copying their code, which I think is not a great idea either.